### ISSUE TITLE
Avatar Magic Burst issues

### ISSUE THREAD

**Issue by rebirthofsword** [EdenServer/community#2263]
### Checklist

- [x ] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x ] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [x ] I have provided my client version in the details. (type /ver into your game window)

### Details
version 30181205_0

when doing a skillchain with an avatar and a player you are not able to then do a magic burst with an avatar.

i haved tested a number of times doing ifrit punch > combo > ifrit fire 2 the logs show the skillchain goes off but no matter the timing the fire 2 will not magic burst

![Rebirthofsword_2019 12 07_224508](https://user-images.githubusercontent.com/58672000/70404522-5a950e80-1a08-11ea-977e-1478e6f8c6dd.png)
![Rebirthofsword_2019 12 07_225548](https://user-images.githubusercontent.com/58672000/70404536-641e7680-1a08-11ea-807c-a11c1e7fafae.png)

**Comment by ShiyoKozuki** [comment]
The way summoner magic bursting works is they have 100% accuracy with no added damage and it doesn't show up in the combat log.

**Comment by Hestichan** [comment]
That's not era accurate, unfortunately. In retail, avatar attacks would 'burst' additional damage. Quoting:

"Since the Dec. 9, 2004 update, Summoners now have the ability to have their Avatar's magical Blood Pacts act as a Magic Burst. Like above, the Blood Pact must be executed after the Skillchain animation and within 5 seconds afterward, and the Avatar's element must match the skillchain element to Magic Burst."

Bottom line: This mechanic from ERA is not working correctly and needs to be fixed, please! It's ruining my parties...

**Comment by 59blargedy** [comment]
validated the code and tested, and these actually all magic burst and add damage based on the burst, just do not have a message. going to close and open a new ticket for clarity since the belief of 'it's ruining my parties' is strong
